qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -65,7 +65,9 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	// Error code for expression part in #case / #is section
 	UnexpectedMemberTypeInCaseSection("Unexpected type `{0}` in `{1}`. Expected `{2}`."),
 	UnexpectedValueInCaseSection("Unexpected value `{0}` in `{1}`. Expected value of type `{2}`."),
+	UnexpectedFirstValueInCaseSection("Unexpected value `{0}`. Expected value of type `{1}` or a valid operator."),

```

Comment:
Instead of introducing a new error code, please use the existing `InvalidOperator` error code.